releases/7.5.0.md

```diff
@@ -61,7 +61,7 @@ None
 
 2. Replace any calls to `django.utils.translation.ugettext()` with `django.utils.translation.gettext()`.
 
-3. In settings.py (or settings_local.py, etc.), remove `USE_L10N`.
+3. In settings.py (or settings_local.py, etc.), remove `USE_L10N`. If present, replace `DEFAULT_FILE_STORAGE` with `STORAGES["default"]["BACKEND"]` and/or `STATICFILES_STORAGE` with `STORAGES["staticfiles"]["BACKEND"]`.
```

Comment:
I think it would be really nice to add a link to the STORAGES section of the Django docs.
https://docs.djangoproject.com/en/4.2/ref/settings/#storages